src/portlet/Portlet.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+import {classNames} from "../utils";
+import {Styles} from "./ClassNames";
+import {Icon, Direction} from "../icon";
+
+type PortletProps = {
+    style?: any;
+    className?: string;
+    header: any;
```

Comment:
In story we are passing string to header prop then why we need any type here ?

Reply:
addressed